front/src/components/ui/ColumnType/File/Input.vue

```diff
@@ -110,7 +110,6 @@ export default {
   },
   watch: {
     attachments: {
-      immediate: true,
```

Comment:
Why did you remove this ?

Reply:
If I remember correctly, the problem here was that if the field was required and empty, an error message was displayed when the form opened, asking the user to enter a value.

So I changed the way it worked to silently validate the field when it was loaded (@hook:mounted), in the same way as the geographic fields.